### PR TITLE
Add my wishes to README.md

### PR DESCRIPTION
**Pull request by Aayushd18**
Wishing Eddie a Happy Birthday!!

Thank you @Pradumnasaraf for this repo. Great idea.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Syntax:
 |[Aditya Joshi](https://twitter.com/aditya__exe) | Happy Birthday Eddie! Loads of love❤️|
 |[Pushkaraj Kulkarni](https://twitter.com/pushkaraj2007)|Happy Birthday Eddie! Have a great year ahead, and Thanks for motivating us to start open source journey.|
 |[Sreekar Shodhan](https://twitter.com/ShodhanSreekar)|Happy Birthday Eddie. I wish all your wishes come true and achieve your goals. Stay Healthy - Stay Happy|
-|[Kalyan Vurugonda](https://twitter.com/VurugondaKalyan)| Happy Birthday Eddie! 🥳🎉 
+|[Kalyan Vurugonda](https://twitter.com/VurugondaKalyan)| Happy Birthday Eddie! 🥳🎉
+|[Aayush Deshmukh](https://twitter.com/aayushdeshmukh)} Happy Birthday Eddie!! 🎉 aka Mr. Opensource!! You make my twitter timeline amazing. |
```
Comment (kartikgajjar7) [inline README.md]: remove " } " and add " | "

**Review by kartikgajjar7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Aayushd18) [reply README.md]: oh lol, I will change

**Review by kartikgajjar7 — APPROVED** [review]
looks good to merge

**Comment by Pradumnasaraf** [comment]
Hey, @kartikgajjar7 again saying please don't approve merge conflict PR.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Pradumnasaraf — APPROVED** [review]
Thank you!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

**Comment by kartikgajjar7** [comment]
> Hey, @kartikgajjar7 again saying please don't approve merge conflict PR.
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix

ohk 👍

**Comment by kartikgajjar7** [comment]
> Hey, @kartikgajjar7 again saying please don't approve merge conflict PR.
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix

hey i m confused what is merge conflict PR ?

**Pull request merged**